02 Local Platform/04 Development Environment/05 Configuration/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "metadata",
+    "values": {
+        "description": "Learn how to configure project-specific properties when working with local platform.",
+        "keywords": "configuration, properties, settings, local project, vs code extension",
```

Comment:
We shouldn't include the page name ("configuration") as a keyword since it's redundant